metadata.json

```diff
@@ -28,6 +28,10 @@
       "name": "puppetlabs-vcsrepo",
       "version_requirement": ">= 1.0.0"
     },
+    {
+      "name": "puppetlabs-yumrepo_core",
```

Comment:
That's not needed, IMHO

Reply:
ah, I see the explanation about gem puppet vs. AIO... I'll remove it in a moment

Thanks!